e2e-tests/test/Helpers/Common.hs

```diff
@@ -18,10 +28,164 @@ cardanoEraToShelleyBasedEra cEra = case cEra of
 
 -- | Make a payment or script address
 makeAddress ::
-  (Either (C.VerificationKey C.PaymentKey) C.ScriptHash) ->
+  Either (C.VerificationKey C.PaymentKey) C.ScriptHash ->
   C.NetworkId ->
   C.Address C.ShelleyAddr
 makeAddress (Left paymentKey) nId =
   C.makeShelleyAddress nId (C.PaymentCredentialByKey $ C.verificationKeyHash paymentKey) C.NoStakeAddress
 makeAddress (Right scriptHash) nId =
   C.makeShelleyAddress nId (C.PaymentCredentialByScript scriptHash) C.NoStakeAddress
+
+-- cardano-api to plutus type conversions:
```

Comment:
I might be easier to maintain if you put it in a different module and re-export it.